packages/samples/test/output/authentication/openapi.yaml

```diff
@@ -52,13 +52,79 @@ components:
     BearerAuth:
       type: http
       scheme: bearer
+      description: >-
```

Comment:
This feels verbose to copy into specs. Should we make it dev-time only?

Reply:
removed it and we can add back later if we feel like it or maybe what's better is when we have the template literal is to have a custom doc with the template values instead.